aurora-git-version-cli/pom.xml

```diff
@@ -53,7 +53,7 @@
     <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-simple</artifactId>
-      <version>1.7.32</version>
+      <version>1.7.30</version>
```

Comment:
Skaper 1.7.32 noe feil?

Reply:
Jeg fikk ikke bygget/testet aurora-git-version lokalt når slf4j-api var eksludert så jeg måtte sette den til samme versjon som kommer med jgit for at enforcer skulle bli fornøyd.